pdc/apps/package/models.py

```diff
@@ -366,13 +364,6 @@ def sync_image_formats_and_types():
         logger.info("Created image type %s" % image_type)
 
 
-@receiver(connection_created)
-def sync_image_formats_and_types_when_connection_created(sender, **kwargs):
-    if 'test' in sys.argv or 'migrate' in sys.argv:
-        return
-    sync_image_formats_and_types()
-
-
```

Comment:
After the first time migration, when we restart the service for some reasons,  don't we need this function?
What error you met? Not sure if the productmd's data will update sometimes.

Reply:
Basically if migrations didn't run yet (db tables don't exist) this would fail.

I would need to change the condition above for checking parameters - but that can break easily in future or with some other parameters.

If productmd is updated while installing new version of PDC, this is not an issue because migrations should be executed and then `sync_image_formats_and_types()` is run.

Not sure if only productmd can be updated without running PDC ansible scripts. 

@lubomir Do you know whether PDC ansible scripts were used last time productmd was updated?

Reply:
I'm not sure about production, but at least stage was updated manually. I don't see it as a big concern anyway: we generally know if there are any changes, and could ask for the migrations to be run as well.

Reply:
If there is no migration at the beginning, I think many errors will raise. So maybe this situation should ignore. 
Not sure how productmd updates, but I think if we keep this trigger here when there is an update request, just need to restart the service (apache).
That's my thinking, but I am also fine with removing these codes as we can do a manually migration.